manifest.json

```
@@ -293,6 +293,22 @@
       "path": "plugins/datetimestamps.lua",
       "version": "0.1"
     },
+    {
+      "description": "Icons font for the devicons plugin.",
+      "id": "font_devicons",
+      "mod_version": "3",
+      "type": "font",
+      "remote": "https://github.com/PerilousBooklet/lite-xl-devicons:e78f2374a1f4de5d0f324af46a07c2f9ffa7ccef",
+      "version": "1.0.0"
+    },
+    {
+      "description": "PerilousBooklet's treeview icons for software developers.",
+      "id": "devicons",
+      "mod_version": "3",
+      "remote": "https://github.com/PerilousBooklet/lite-xl-devicons:e78f2374a1f4de5d0f324af46a07c2f9ffa7ccef",
+      "version": "1.0.0",
+      "dependencies": { "font_devicons": {} }
```

```suggestion
      "version": "1.0.0"
```
No need to specify dependencies here.